commons/src/test/java/org/eclipse/kapua/commons/event/ServiceEventClientConfigurationTest.java

```diff
@@ -0,0 +1,91 @@
+
+    ServiceEventBusListener serviceEventBusListener;
+
+    @Test
+    public void constructorRegularTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration("address", "subscriberName", serviceEventBusListener);
+        assertEquals("not_equals", "address", serviceEventClientConfiguration.getAddress());
+        assertEquals("not_equals", "subscriberName", serviceEventClientConfiguration.getClientName());
+        assertEquals("not_equals", serviceEventBusListener, serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration(null , null, null);
+        assertNull("not_null", serviceEventClientConfiguration.getAddress());
+        assertNull("not_null", serviceEventClientConfiguration.getClientName());
+        assertNull("not_null", serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorAddressNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration(null , "subscriberName", serviceEventBusListener);
+        assertNull("not_null", serviceEventClientConfiguration.getAddress());
+        assertEquals("not_equals", "subscriberName", serviceEventClientConfiguration.getClientName());
+        assertEquals("not_equals", serviceEventBusListener, serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorClientNameNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration("address", null, serviceEventBusListener);
+        assertEquals("not_equals", "address", serviceEventClientConfiguration.getAddress());
+        assertNull("not_null", serviceEventClientConfiguration.getClientName());
+        assertEquals("not_equals", serviceEventBusListener, serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorServiceEventNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration("address", "subscriberName", null);
+        assertEquals("not_equals", "address", serviceEventClientConfiguration.getAddress());
+        assertEquals("not_equals", "subscriberName", serviceEventClientConfiguration.getClientName());
+        assertNull("not_null", serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorAddressAndNameNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration(null, null, serviceEventBusListener);
+        assertNull("not_null", serviceEventClientConfiguration.getAddress());
+        assertNull("not_null", serviceEventClientConfiguration.getClientName());
```

Comment:
Please change the message to: "Expected value should be null" or something similar. "not_null" is not a valid comment.